web/src/createProjectByPattern.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import UploadByLink from './createProjectView.js';
+
+
+export default class UploadByPattern extends React.Component {
+  render(){
+    return(
+      <div className="content">
+        <div className="headerCreation">Create New Project</div>
+        <h3 className="caption">Project Name</h3>
+        <input className="enterValue" type="text"  ></input>
+        <h3 className="caption">Author Name</h3>
+        <input className="enterValue" type="text"  ></input>
+        <h3 className="caption">Team</h3>
+        <select className="dropdown" name="Select Team" size="1">
+          <option></option>
+          <option></option>
+          <option></option>
+          <option></option>
+          <option></option>
+        </select>
+        <h3 className="caption">Status</h3>
+        <select className="dropdown" name="Select Team" size="1">
+          <option>finished</option>
+          <option>pending</option>
+          <option>reviewed</option>
+        </select>
+        <h3 className="caption">Description</h3>
+        <textarea className="enterDescription" type="text"  ></textarea>
```

Comment:
just as a advice, you can also do this:
<textarea className="enterDescription" type="text" /> 
😉